cilium/CFP-30984-dns-proxy-ha.md

```diff
@@ -85,67 +87,114 @@ message Result {
 }
 ```
 
-Method : GetEndpointMetadataByIP
-
-Request : 
-```
-message FQDNSelector {
-  string ip = 1;
-}
-```
-Response : 
-```
-message Result {
-    uint32 endpoint_id = 1;
-    uint32 identity = 2;
-}
-```
-
 ### Tracking policy updates to SDP instances
 
 Since SDP can be deployed out of band and users can choose to completely disable built-in proxy to run multiple instances of SDP, agent should be prepared to handle multiple instances. In order to ensure all instances  have upto date policy revisions, agent will maintain a mapping of ACKed policy revision numbers against stream ID.
-
+Since policy revision numbers from Cilium are reset when the local Cilium instance restarts, we need to unconditionally send the policy revision number to the SDP instance on agent restart.
```

Comment:
We need to notify SDP of the policy though right ? SDP itself has no real use of revision numbers.

Reply:
That's right. I will update it to send policy updates and not just revision number.